src/modules/sih/sih.cpp

```diff
@@ -430,6 +516,22 @@ void Sih::publish_sih()
 	_gpos_gt_pub.publish(_gpos_gt);
 }
 
+// quaternion exponential as defined in [3]
+Quatf Sih::expq(matrix::Vector3f u)
```

Comment:
```suggestion
Quatf Sih::expq(const matrix::Vector3f& u)
```

Reply:
is it about saving memory by avoiding to reallocate?

Reply:
It's to avoid having to copy the vector when calling the method.